android/app/build.gradle

```diff
@@ -127,9 +127,9 @@ android {
     }
 }
 
-// Excludes duplicate inclusions of OkHttp - from SalesforceSDK and React. Forces OkHttp 3.4.1 to be used.
+// Excludes duplicate inclusions of OkHttp - from SalesforceSDK and React. Forces OkHttp 3.6.0 to be used.
 configurations.all {
-    resolutionStrategy.force 'com.squareup.okhttp3:okhttp:3.4.1'
+    resolutionStrategy.force 'com.squareup.okhttp3:okhttp:3.6.0'
```

Comment:
@brandonpage Does it work if you remove this block altogether, i.e. does it work with `3.9.0` that we use?

Reply:
Yes, I believe so (I had deleted it entirely before because 3.4.1 caused a problem).   I will check and update if it's fine.

Reply:
Yup - let's remove the block so it uses `3.9.0`.